### DISCUSSION TITLE
Welcome to 32health-backend Discussions!

### DISCUSSION THREAD

**Discussion by crisps0914** [crisps0914/32health-backend#3] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by crisps0914** [comment]
I'm sorry, but this text doesn't provide me any context about your issue. Could you please give me more details on what you are trying to do and the code that is causing the problem?

**Comment by shadow973** [comment, marked as answer]
I'm sorry, but this text doesn't provide me any context about your issue. Could you please give me more details on what you are trying to do and the code that is causing the problem?
